frontend/styles/components/calendar/mobile/DayPortraitView.module.scss

```
@@ -1,14 +1,10 @@
-@import '../../Variables.module.scss';
+@import '../../../Variables.module.scss';
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)